CHANGELOG.md

```diff
@@ -0,0 +1,310 @@
+# Changelog
+
+Notable changes to this project are documented in this file. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+Breaking changes (😱!!!):
```

Comment:
I noticed a previous version of this PR removed the emoji.
I thought this was for compatibility reasons. I have no preference either way.

Reply:
Thanks for catching this. I updated my code, but didn't recompile it. Whoops!